homeassistant/components/actiontec/device_tracker.py

```diff
@@ -33,43 +31,40 @@
 )
 
 
-def get_scanner(hass, config):
+def get_scanner(
+    hass: HomeAssistant, config: ConfigType
+) -> ActiontecDeviceScanner | None:
```

Comment:
This should return a `DeviceScanner` instance. The interface requires that class, so setting the platform specific class as return value type could let the platform specific class avoid inheriting `DeviceScanner` which would be an error.

Reply:
just to be sure - should I file a small PR for that?

Reply:
Yes, please 🙂